frontend/app/src/components/asset-manager/ManagedAssetForm.vue

```diff
@@ -249,14 +249,17 @@ const save = async () => {
     await get(assetIconFormRef)?.saveIcon(newIdentifier);
     return true;
   } catch (e: any) {
-    const message = deserializeApiErrorMessage(e.message) as any;
-    if (!message) {
-      setMessage({
-        title: tc('asset_form.underlying_tokens'),
-        description: e.message
-      });
-    } else {
-      handleError(message);
+    if (e instanceof ApiValidationError) {
+      const errors = e.getValidationErrors({});
+
+      if (typeof errors === 'string') {
+        setMessage({
+          title: tc('asset_form.underlying_tokens'),
+          description: errors
+        });
+      } else {
+        handleError(errors);
+      }
     }
```

Comment:
Again this seems like it is suppressing all errors unless it is an ApiValidationError, which is not what the code before was doing before.